docs(ai): Iterate on koji agent (#15904)
microsoft/CBL-Mariner: README.md

| `azl-review-component` | Audit a component for hygiene and best practices. |
| `azl-migrate-component` | Move an inline entry to a dedicated `.comp.toml` file. |

### Agents

> In addition to prompts, the repo includes **chat agents** that can be selected from the chat participant dropdown (the model/agent picker at the top of the Copilot Chat panel). Select the agent, then type your query:

Author: ```suggestion
In addition to prompts, the repo includes **chat agents** that can be selected from the chat participant dropdown (the model/agent next to the Copilot Chat panel). Select the agent, then type your query:
```
